spec/feedback_runner_spec.rb

```diff
@@ -66,7 +67,56 @@ class Foo {
 
       it {expect(feedback).to include("Hay un método que debería retornar algo, pero no está retornando nada. ¡Revisá bien tu código!")}
     end
-    context 'missing return statement' do
+
+    context 'missing method with type' do
+      let(:request) {req(%q{
+    class Golondrina {
+    }}, %q{
+      public void testFoo(){
+        Golondrina golondrina = new Golondrina();
+        golondrina.reanimarConUnChocolate();
+      }
+    })}
+
+      it {expect(feedback).to include("* No se encontró la definición de el método `reanimarConUnChocolate()` en la variable `golondrina` de tipo `Golondrina`")}
```

Comment:
del

Reply:
Hmm yes, but I think is not that easy. We've the same prolem with 'a'/'an' in english...

Reply:
Maybe we could use `la definición de `el método reanimarConUnChocolate()`?